File: 백준/Bronze/32905. RACI/RACI.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String args[]) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer st;
        
        st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken());
        int m = Integer.parseInt(st.nextToken());
        
        boolean isValid = true;
        
        for (int i = 0; i < n; i++) {
            int accountableCount = 0;
            String[] task = br.readLine().split(" ");
            for (String person : task) {
                if (person.equals("A")) accountableCount++;
            }
            if (accountableCount != 1) {
                isValid = false;
                break;
            }
        }
        bw.write(isValid ? "Yes" : "No");
        bw.flush();
        br.close();
        bw.close();
    }
} 

File: 백준/Bronze/32905. RACI/RACI.cc
#include <iostream>

using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(NULL);

    int n, m;
    cin >> n >> m;

    bool is_valid = true;
    
    for (int i = 0; i < n; i++) {
        int accountable_count = 0;
        for (int j = 0; j < m; j++) {
            char person;
            cin >> person;
            if (person == 'A') accountable_count++;
        }

        if (accountable_count != 1) {
            is_valid = false;
            break;
        }
    }

    cout << (is_valid ? "Yes" : "No") << '\n';

    return (0);
}
